book/http.md

```diff
@@ -41,8 +41,8 @@ the URL, like ports, queries, and fragments, which we'll see later.
 
 From a URL, the browser can start the process of downloading the web
 page. The browser first asks the local operating system (OS) to put it in touch with the
-*server* described by the *host name*. The OS then talks to a *DNS*
-(Domain Name System)
+*server* described by the *host name*. The OS then talks to a
+Domain Name System (*DNS*)
```

Comment:
I think in this case we make "Domain Name System" italics, then, and keep the acronym in upright text?